tool/travis.sh

```diff
@@ -59,7 +59,7 @@ elif [ "$BOT" = "test_dart2js" ]; then
     pub global activate webdev
 
     WEBDEV_RELEASE=true pub run test --reporter expanded --exclude-tags useFlutterSdk
-    WEBDEV_RELEASE=true pub run test --reporter expanded --exclude-tags useFlutterSdk --platform chrome-no-sandbox
+    WEBDEV_RELEASE=true pub run build_runner test -- --reporter expanded --exclude-tags useFlutterSdk --platform chrome-no-sandbox
```

Comment:
I also noticed this `WEBDEV_RELEASE` var doesn't do anything for web tests, so I am not sure why it is supplied here.

Reply:
given it doesn't do anything we should remove it for clarity.

Reply:
should this have the "-r" flag to run under dart2js?
Go ahead and remove any flags that are obsolete so we don't cargo cult copy them to more places.